[PXCT-390] Updated cloud docs for device priority (#2410)
arduino/docs-content: content/arduino-cloud/03.cloud-interface/00.sketches/sketches.md

}
```

> There are a few different options when it comes to synchronization of your cloud sketch:

Reviewer: ```suggestion
There are a few different options available for synchronizing your sketch with the Arduino Cloud:
```
